src/shim/crypto.rs

```diff
@@ -88,7 +88,7 @@ impl Signature {
 
     /// Checks if a signature is valid given data and address.
     pub fn verify(&self, data: &[u8], addr: &crate::shim::address::Address) -> Result<(), String> {
-        use fvm_shared3::crypto::signature::ops::{verify_bls_sig, verify_secp256k1_sig};
+        use fvm_shared4::crypto::signature::ops::{verify_bls_sig, verify_secp256k1_sig};
```

Comment:
should we rather use `fvm_shared_latest` here?

Reply:
Fixed.